game-core/src/main/java/games/strategy/triplea/attachments/TechAbilityAttachment.java

```diff
@@ -6,12 +6,18 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.function.Function;
+
+import com.google.common.collect.ImmutableMap;
+
 import java.util.Set;
```

Comment:
Checkstyle: "Import statement for 'java.util.Set' is in the wrong order. Should be in the 'STANDARD_JAVA_PACKAGE' 
 group, expecting not assigned imports on this line."